types/next/test/next-router-tests.tsx

```diff
@@ -79,16 +79,18 @@ class TestComponent extends React.Component<TestComponentProps & WithRouterProps
 
     constructor(props: TestComponentProps & WithRouterProps) {
         super(props);
-        props.router.ready(() => {
-            this.setState({ ready: true });
-        });
+        if (props.router) {
+            props.router.ready(() => {
+                this.setState({ ready: true });
+            });
+        }
     }
 
     render() {
         return (
             <div>
                 <h1>{this.state.ready ? 'Ready' : 'Not Ready'}</h1>
-                <h2>Route: {this.props.router.route}</h2>
+                <h2>Route: { this.props.router ? this.props.router.route : "" }</h2>
```

Comment:
Let's remove the spaces between the curly brackets now that we know the CI is fixed.

Reply:
Done. Thanks.